src/backend/core/api/viewsets.py

```diff
@@ -233,6 +248,90 @@ def perform_create(self, serializer):
             role=models.RoleChoices.OWNER,
         )
 
+    @decorators.action(
+        detail=True,
+        methods=["post"],
+        url_path="start-recording",
+        permission_classes=[
+            permissions.HasPrivilegesOnRoom,
+            permissions.IsRecordingEnabled,
+        ],
+    )
+    def start_room_recording(self, request, pk=None):  # pylint: disable=unused-argument
+        """Start recording a room."""
+
+        hrid = request.data.get("hrid")
+        if hrid is None or not hrid.strip():
+            return drf_response.Response(
+                {"error": "Recording hrid is required."},
+                status=drf_status.HTTP_400_BAD_REQUEST,
+            )
+
+        # todo - check value of the hrid
```

Comment:
should I check the value of the hrid? 
Calling `worker_service = get_worker_service(hrid=recording.worker_type)` with an invalid hrid would raise an error.